### PR TITLE
fiona/ fix banner in mt5 dashboard on mobile

### PR DESCRIPTION
**Pull request by fiona-deriv**

### CODE REVIEW

File: src/javascript/app/base/socket_general.js [hunk 1]
```diff
@@ -162,9 +161,6 @@ const BinarySocketGeneral = (() => {
                 }
                 break;
             }
-            case 'RateLimit':
```
Comment (mahdiaryayi-fs) [inline src/javascript/app/base/socket_general.js:165]: Are you removing the whole functionality of the RateLimit alert or is it appearing somewhere else?
Don't we need this functionality anymore?

Comment (fiona-deriv) [reply src/javascript/app/base/socket_general.js:165]: @mahdiaryayi-fs rate limit alert for mt5 seems to be appearing somewhere else

Comment (negarn) [reply src/javascript/app/base/socket_general.js:165]: @fiona-deriv then we should remove that other one and keep this. this is used for our whole website.

Comment (fiona-deriv) [reply src/javascript/app/base/socket_general.js:165]: okay thanks, noted

**Pull request closed without merging**